spec/index.spec.js

```diff
@@ -462,13 +462,34 @@ describe('server', () => {
       .then(done);
   });
 
+  it('fails if default limit is negative', done => {
+    reconfigureServer({ defaultLimit: -1 }).catch(error => {
+      expect(error).toEqual('Default limit must be a value greater than 0.');
+      done();
+    });
+  });
```

Comment:
prefer tests to be formatted as:

```suggestion
  it('fails if default limit is negative', async () => {
    await expectAsync(reconfigureServer({ defaultLimit: -1 })).toBeRejectedWith('Default limit must be a value greater than 0.'));
  });
```